streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/worker/JobWorker.java

```diff
@@ -124,6 +121,10 @@ public void onReaderMessage(byte[] buffer) {
    * and receive result from this actor
    */
   public byte[] onReaderMessageSync(byte[] buffer) {
+    if (transferHandler == null) {
+      // new byte[4] means not ready.
+      return new byte[4];
```

Comment:
This is too hacky. Why not use a constant?

Reply:
fixed